src/site/markdown/server/manage-webadmin.md

```diff
@@ -440,7 +440,7 @@ The scheduled task will have the following type `full-reindexing` and the follow
 Given `bbdb69c9-082a-44b0-a85a-6e33e74287a5` being a taskId generated for a reIndexing tasks
 
 ```
-curl -XPOST http://ip:port/mailboxes?task=reIndex&reIndexFailedMessagesOf=bbdb69c9-082a-44b0-a85a-6e33e74287a5
+curl -XPOST http://ip:port/mailboxes?task=reIndex\&reIndexFailedMessagesOf=bbdb69c9-082a-44b0-a85a-6e33e74287a5
```

Comment:
for this commit, (s/&/\&/), I would prefer that you put quotes (double or single, as you wish) around the url. It would allow an easier copy paste to other tools than sh.
BTW that's not curl that need this, that's (ba)sh.

Reply:
That's why the commit title said `curl [command examples] needs **bash** escaping`